TELCODOCS#1759: Adding must gather image for LVM Storage (#71114)
openshift/openshift-docs: modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc

:_mod-docs-content-type: PROCEDURE
[id="lvms-unstalling-lvms-with-web-console_{context}"]

> = Uninstalling {lvms} installed using the web console

Reviewer: ```suggestion
= Uninstalling {lvms} installed using the {product-title} web console
```

Author: I have updated the title to "Uninstalling {lvms} using the web console"
As users can install LVMS using both the web console and the CLI. It's not a restriction that this proc is applicable only if LVMS was installed through the web console.

> You can uninstall {lvms} using the OpenShift Container Platform web console.

Reviewer: ```suggestion
You can uninstall {lvms} using the {product-title} web console.
```
Missed this on my first time through @sr1kar99

Author: Updated as per the suggestion.

[id="lvms-unstalling-lvms-with-web-console_{context}"]
= Uninstalling {lvms} installed using the OpenShift Web Console

> You can uninstall {lvms} using the Red Hat OpenShift Container Platform Web Console.

Reviewer: Please add a {nbsp} between Red and Hat. This prevents the company name from breaking across lines in the docs.

Reviewer 2: Also, I will add that we do not capitalize web console.
```suggestion
You can uninstall {lvms} using the Red Hat OpenShift Container Platform web console.
```
Also should be lower case in the module title as well.

Reviewer 2: I also would like to add that you might want to consider revising some of the prerequisites and steps to this procedure to adhere more to the style guide.  To list a few for example:

- We should avoid directional language such as "right-hand side" used on lines 23 and 25. 
- s/drop-down menu/list
- "You have access to the cluster using an account with `cluster-admin` permissions." should be first prereq because it is essential to do the rest of the prereqs and the procedure.
- Step 1, consider revising to remove "scroll to *LVM Storage* or type `LVM Storage` into the *Filter by name* to find and click on it." you can just say "click *LVM Storage*"
- Step 2,3, and 4 could be one step: "Click the *LVMCluster* tab and select *Delete LVMCluster* and *Uninstall Operator* from the *Actions* list.
- Last step: "Click *Remove*"

I know it is out of scope of this PR, so you could do it here or create a new issue for it. Feel free to ping me if you have questions. 

Ps this confluence page is a good resource to help with UI elements: https://docs.engineering.redhat.com/pages/resumedraft.action?draftId=380770273&draftShareId=71b5b07a-7293-46dd-ae22-f71ebbc24066&

Author: @mjpytlak 
Thank you for the review.
I've removed "Red Hat" as it is not used in other modules of the assembly.
To maintain consistency, the line and the title have been updated.
PTAL.
